static/src/javascripts/projects/common/modules/commercial/build-page-targeting.spec.js

```diff
@@ -72,9 +76,18 @@ const tcfNullConsentMock = (callback): void =>
     callback({ '1': null, '2': null, '3': null, '4': null, '5': null });
```

Comment:
Should this not be an empty object?
```suggestion
    callback({});
```

Reply:
that's for tcfv1 check `tcfv2WithoutConsentMock` for tcfv2